doc/index.rst

```diff
@@ -67,9 +67,10 @@ CCArchiver
 Overview
 ========
 
-[This section is to present an overview of the CSC.
-This should include a top-level description of the primary use-case(s) as well as any pertinent information.
-Example information may be link(s) to the higher-level classes which may be used to operate it, or mention of other CSCs (with links) that it operates in concert with.]
+The Data Management CCArchiver coordinates Commissioning Camera 
+archiving activities for the Vera C. Rubin Observatory.  This service 
+interacts with other services coordinate retrieval and archival of
+image data from the ComCam data acquisition system.
```

Comment:
This feels a little light on top-level detail. You have details in the first few sentences of the dev guide that offer a little more insight into things (mentions which services it interacts with etc). I suggest moving the top few sentences here. 
Then maybe 1 sentence about how the user does not directly interact with the CSC. 
So far, I think the only thing the user really directly listens for is the event when the image is in the OODS? So maybe add something to that affect.

Reply:
Added